certified-connectors/Dynatrace/apiProperties.json

```diff
@@ -0,0 +1,57 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      },
+      "tenantUrl": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "Dynatrace Tenant",
+          "description": "The url for your tenant. (abc123.dynatrace.com)",
+          "tooltip": "Provide your Dynatrace Tenant",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#242126",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "TenantURL",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('tenantUrl')"
+        }
+      },
+      {
+        "templateId": "setqueryparameter",
+        "title": "GetProblemQueryParameters",
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "from",
+          "x-ms-apimTemplateParameter.value": "now-1w",
+          "x-ms-apimTemplateParameter.existsAction": "override",
+          "x-ms-apimTemplate-operationName": [
+            "GetProblems"
+          ]
+        }
+      }
+    ],
+    "publisher": "Swinton, Mandy",
```

Comment:
@mandy-swinton please make sure the publisher is specified according to [this template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json). It should be the name of company or organization.

Reply:
I have updated the publisher